eng/Packages.Data.props

```
@@ -106,6 +106,7 @@
     <PackageReference Update="Microsoft.Azure.WebPubSub.Common" Version="1.1.0" />
     <PackageReference Update="Microsoft.Identity.Client" Version="4.46.0" />
     <PackageReference Update="Microsoft.Identity.Client.Extensions.Msal" Version="2.23.0" />
+    <PackageReference Update="RL.Net"  VersionOverride="0.2.0"/>
```

```suggestion
    <PackageReference Update="RL.Net" Version="0.2.0"/>
```

Is this package dependency approved by ArchBoard? /cc @tg-msft @KrzysztofCwalina